requirements/software.md

```diff
@@ -0,0 +1,99 @@
+# Software Requirements
+
+* ### Pharmacy Files
+
+    It is likely that a software project has several different engineering
+    documents associated with it such as software requirements,
+    architecture design, and so forth. Each of these documents may have a
+    corresponding Rx file that Howser can use to validate their
+    conformance to a prescribed format. In a continuous integration
+    environment, it makes sense to have those validations be performed
+    automatically as a step in the build process. In order for that to
+    occur, there must be a way to specify which documents in the project
+    should be checked for conformance and which prescriptions each of
+    those documents should be checked against. This is the motivation for
+    introducing a `pharmacy.toml` file as a way to specify this configuration
+    on a per-project basis. Pharmacy files are written in toml and checked
+    into the repository along with the source code. They specify a series
+    of key value pairs that correspond to prescription files and the
+    documents that are to be checked with them.
+    
+    **Rx spec** = A DSL for constraining the content of Markdown 	documents
+
+    **prescription file (.rx)** =  A document written using the Rx spec DSL
+
+    **Pharmacy file (.toml)** = A list of documents to validate paired with the 	prescription they’re validated against
+
+    ### Acceptance Requirements
+
+    #### Background
+    
+    * **Given** that there exists some valid prescription files arbitrarily located within the project
+        * **And** the prescription files are divided into groups "matching" and "mis-matching"
+        * **And** for each of the prescription files in the "match" group there exists a corresponding markdown file somewhere in the project that matches the prescription
+        * **And** for each of the prescription files in the "mis-matching" group there exists a corresponding markdown file somewhere in the project that does not match the prescription
+        * **And** there exists a "pharmacy.toml" file 
+        * **And** the pharmacy file contains a toml section labeled "Specs"
+        * **And** for each prescription file there exists a toml key-value pair in the "Specs" section of the pharmacy file with filename as key and markdown filename as value
```

Comment:
To be more precise, this could be refined to "with Rx spec filename as key and markdown filename as value", otherwise it is ambiguous what filename is the key.